README.md

```diff
@@ -55,27 +42,63 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
   }
   ```
 
-15. There MUST NOT be any assignment or modification of variables in the expressions of `if` statements.
+13. There MUST NOT be any assignment or modification of variables in the expressions of `if` statements.
 
   ```php
   if ($response = $request->getResponse()) {
     // not allowed
   }
   ```
-16. Abstract class names MUST begin with `Abstract`.
+14. Abstract class names MUST begin with `Abstract`.
 
   ```php
   abstract class AbstractGenerator {
   }
   ```
-17. Interface names MUST end with `Interface`.
+15. Interface names MUST end with `Interface`.
 
   ```php
   interface GeneratorInterface {
   }
   ```
 
-18. Doc block types for scalar values MUST be one of: `bool` (not boolean), `int` (not integer), `string`, `float` (not double)
+### PHP DocBlock
+1. DocBlock for functions / methods MUST exist where the function / method has arguments or a return value and MUST use the appropriate tags (@param, @return) to denote that. Developers making modifications to a function / method are tasked with ensuring that the DocBlock is up-to-date.
+2. Functions / methods / classes that are sufficiently complex (i.e. not self-documenting) SHOULD have a DocBlock explaining how to use the function / method / class.
+3. DocBlock presenting the type MUST be present for class member variables.
+4. When a description is necessary for class member variables, the DocBlock MUST be multiline:
+
+  ```php
+  class Foo
+  {
+      /** @var int */
+      protected $id;
+      
+      /**
+       * The Bar used to fight the foo
+       * @var Bar
```

Comment:
@wpillar Good point. I've mostly been putting a blank line in between, but never thought about this being a requirement. It might be clearer if we make it one. What's your thought?